source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm

```diff
@@ -283,7 +315,18 @@ - (void)addTasksToConcurrentQueue:(std::vector<std::shared_ptr<BaseCardElement>>
 - (void)loadImagesForActionsAndCheckIfAllActionsHaveIconImages:(std::vector<std::shared_ptr<BaseActionElement>> const &)actions hostconfig:(ACOHostConfig *)hostConfig;
 {
     for(auto &action : actions){
-        if(action->GetIconUrl().size()) {
+        if(!action->GetIconUrl().empty()) {
+            ObserverActionBlock observerAction =
```

Comment:
Is it intentional that this observerAction is not being used? The render methods for images above use their ObserverActionBlocks for subsequent -loadImageAccordingToResourceResolverIF calls, but this is just calling -loadImage.

Reply:
yes, this is intentional. the support has not been added for actions, yet. 

---
In reply to: [237271207](https://github.com/Microsoft/AdaptiveCards/pull/2169#discussion_r237271207) [](ancestors = 237271207)